<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*VERTICAL SCROLLBAR*/
QScrollBar:vertical{
	boder: none;
	background-color: rgb(59,59,90);
	width: 14px;
	margin: 15px 0 15px 0;
	boder-radius: 0px;
}

QScrollBar::handle:vertical{
	background-color: rgb(80,80,120);	
	min-height: 30px;
	border-radius: 7px;
}

QScrollBar::handle:vertical:hover{
	background-color: rgb(255, 0, 127);
}

QScrollBar::handle:vertical:pressed{
	background-color: rgb(185, 0, 92);
}
/*BIN TOP- SCROLLBAR*/
QScrollBar::sub-line:vertical{
	border: none;
	background-color: rgb(59,59,90);
	height: 15px;
	border-top-left-radius: 7px;
	border-top-right-radius: 7px;	
	subcontrol-position: top;
	subcontrol-origin: margin;
}


QScrollBar::sub-line:vertical:hover {
	backgroung-color: rgb(255,0,127);
}
QScrollBar::sub-line:vertical:pressed {
	background-color: rgb(185, 0, 92);
}

/*BIN BOTTOM	- SCROLLBAR*/
QScrollBar::add-line:vertical{
	border: none;
	background-color: rgb(59,59,90);
	height: 15px;
	border-bottom-left-radius: 7px;
	border-bottom-right-radius: 7px;	
	subcontrol-position: bottom;
	subcontrol-origin: margin;
}


QScrollBar::add-line:vertical:hover {
	backgroung-color: rgb(255,0,127);
}
QScrollBar::add-line:vertical:pressed {
	background-color: rgb(185, 0, 92);
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
	background: none;
}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background: none;
}






</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 58, 89); </string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>901</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar:vertical{
	border: none;
	background: rgb(56,56,85);
} </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>885</width>
       <height>1218</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame_right">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>1200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(56, 58, 89);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="list_infor">
           <property name="font">
            <font>
             <family>Segoe UI Historic</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
             <stylestrategy>PreferDefault</stylestrategy>
             <kerning>true</kerning>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_left">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>601</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(109, 113, 173);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#fbbeff;&quot;&gt;IP: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#fbbeff;&quot;&gt;PORT: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_port">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(230, 179, 255);</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_ip">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
